File: DS-linkedlist.cpp
#include <bits/stdc++.h>
#include <iostream>
using namespace std;

class Node {
   public:
  int value;
  Node* next;
};

int main() {
  Node* head;
  Node* one = NULL;
  Node* two = NULL;
  Node* three = NULL;

  one = new Node();
  two = new Node();
  three = new Node();

  one->value = 3;
  two->value = 6;
  three->value = 9;

  one->next = two;
  two->next = three;
  three->next = NULL;

  head = one;
  while (head != NULL) {
    cout << head->value;
    head = head->next;
  }
}

File: DS-linkedlist.py
class Node:
    def __init__(self, item):
        self.item = item
        self.next = None

class Linked:

    def __init__(self):
        self.head = None


if __name__ == '__main__':

    linked_list = Linked()

    linked_list.head = Node(4)
    second = Node(5)
    third = Node(6)

    linked_list.head.next = second
    second.next = third

    #
    while linked_list.head != None:
        print(linked_list.head.item, end=" ")
        linked_list.head = linked_list.head.next
